app/views/accounts/_eligible_for_funding_details.html.erb

```diff
@@ -0,0 +1,29 @@
+<p class="govuk-!-margin-top-2 govuk-!-margin-bottom-0">
+  This means that if <%= application.lead_provider.name %> confirms that you have a scholarship funded space, you will not need to pay for the course fees.
+</p>
+<p class="govuk-!-margin-top-2">
+  Contact <%= application.lead_provider.name %> to check if they can offer you a scholarship funded space, if you have not done so already.
+</p>
+
+<% def details_text %>
+  <p class="govuk-!-margin-bottom-2">You can:</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>
+      register with this provider again at a later date (courses are run twice a year,
+      except for the Early headship coaching offer which varies by provider and is available year-round)
+    </li>
+    <li>
+      register with a different provider
+    </li>
+    <li>
+      explore other payment options, including paying for the course yourself or checking
+      if your workplace or trust can cover the cost
+    </li>
+  </ul>
+<% end %>
+
+<%= govuk_details(
+  summary_text: "What if no funded spaces are available with my chosen provider?",
+  text: details_text.html_safe
+) %>
```

Comment:
Ah, the component takes a block, this should work 🙂 

```suggestion
<%= govuk_details(summary_text: "What if no funded spaces are available with my chosen provider?") do %>
  <p class="govuk-!-margin-bottom-2">You can:</p>

  <ul class="govuk-list govuk-list--bullet">
    <li>
      register with this provider again at a later date (courses are run twice a year,
      except for the Early headship coaching offer which varies by provider and is available year-round)
    </li>
    <li>
      register with a different provider
    </li>
    <li>
      explore other payment options, including paying for the course yourself or checking
      if your workplace or trust can cover the cost
    </li>
  </ul>
<% end %>
```

Reply:
It just means that if/when the [Details component](https://design-system.service.gov.uk/components/details/) changes upstream we don't have to go and find all the hardcoded instances of it and fix them manually.